README.md

```diff
@@ -1,2 +1,7 @@
 # FEGA-Norway
 FEGA-Norway Mono repo
+
+## Getting Started
+For guidance, check out the instructions in the [e2eTests folder](e2eTests/README.md) of the repository
+
+If you run into any problems during setup, there's a troubleshooting guide [available here](e2eTests/README.md#troubleshooting).
```

Comment:
```suggestion
If you run into any problems during setup, there's a troubleshooting guide [available here](e2eTests/README.md#troubleshooting).
```
We can remove this as the documentation is moved to https://github.com/ELIXIR-NO/FEGA-Norway/wiki/Local-Setup